### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by vic778**
-Added the index.html
-Added the style.css
-Added the README.md

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100;300;400;500;700&family=Roboto:wght@100;300;400;500;700&family=Inter:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+    <title>Victor Portfolio</title>
+  </head>
+  <body>
+    <!-- Header Section -->
+    <header class="header-bar d-flex">
+      <a class="header-title" href="#">victor barh</a>
+      <a class="menu-button" href="#" onclick="toggle()">
+        <img id="hamburger-btn" src="assets/hamburger.svg" alt="hamburger">
+      </a>
```
Comment (Mihndim2020) [inline index.html:27]: - [ ] Please hamburger button should be within a `<nav>` tag as it is a requirement for this project and it makes your html more [semantic](https://www.w3schools.com/html/html5_semantic_elements.asp). Kindly fix that.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Linters are passing
:heavy_check_mark: Good design implementation

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION


# Status: Requires Changes ♻️
_Check the comments below under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (vic778) [reply index.html]: hello, sir I put the hamburger in the nav as you suggested. you can look then

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @vic778, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**